spec/services/catalog/notify_approval_request_spec.rb

```diff
@@ -29,6 +29,10 @@
           expect(approval_request.reason).to eq("because")
         end
 
+        it "updates the request completed at time" do
+          expect(approval_request.request_completed_at).to be_within(1.second).of(Time.now.utc)
```

Comment:
Should we just validate that `request_completed_at` is set? The 1.second time different could be problematic in some cases and the code simply updates the time to `Time.now.utc` with no concern for how fast that happens.

Reply:
I mainly just used the "within 1 second" because I didn't want to introduce timecop to freeze the time. I can't think of why the spec would take longer than 1 second to complete, we could bump it to `2.seconds` if you think that gives more leeway while still staying true to the essence of what the spec is testing?